Configure

```diff
@@ -422,6 +422,7 @@ my @disablables = (
     "ubsan",
     "ui-console",
     "unit-test",
+	"uplink",
```

Comment:
Indent with spaces, please.

Reply:
Also, please document this configuration option in `INSTALL` somewhere under the title "Configuration Options".  Since it's enabled by default where applicable, it should be presented as `no-uplink` in `INSTALL`.

Reply:
Finally, a note in `CHANGES` would be a good thing.  We haven't been consistent with that, but I try to remind us all when I think of it.

Reply:
> Indent with spaces, please.

> Also, please document this configuration option in INSTALL somewhere under the title "Configuration Options".  Since it's enabled by default where applicable, it should be presented as no-uplink in INSTALL.

> Finally, a note in CHANGES would be a good thing.  We haven't been consistent with that, but I try to remind us all when I think of it.

I have tried my best to follow your suggestions.